conf/app.routes

```diff
@@ -72,4 +72,6 @@ GET        /account-home/close-account-are-you-sure         @uk.gov.hmrc.helptos
 GET        /enter-uk-bank-details                           @uk.gov.hmrc.helptosavefrontend.controllers.BankAccountController.getBankDetailsPage
 POST       /enter-uk-bank-details                           @uk.gov.hmrc.helptosavefrontend.controllers.BankAccountController.submitBankDetails
 
-GET        /check-details                                   @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.checkDetails
\ No newline at end of file
+GET        /check-details                                   @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.checkDetails
+GET        /change-bank-details                             @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.changeBankDetails
+GET        /change-email-address                            @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.changeEmail
```

Comment:
this seems in-consistent as any changes to emails or bank-details should go to appropriate controllers

Reply:
I wanted to keep it close to check-details as they're triggered from that page

Reply:
the email controllers getting big enough as it is!

Reply:
hmm , in that case we will split email controller logic into two classes one for Digital and one for DE